index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
+            <p>
+                Other JOSE header parameters and claim names not specified herein can be used in addition if their
+                use is not explicitly discouraged.
+            </p>
+
+            <pre class="example nohighlight" title="JWT header of a JWT-based verifiable credential using JWS as a proof (non-normative)">
+            {
+                "alg": "RS256",
+                "typ": "JWT",
+                "kid": "did:example:abfe13f712120431c276e12ecab#keys-1"
+            }
+            </pre>
+
+            <p class="issue">
+                Add a short description explaining the example above.
+            </p>
+
+            <pre class="example nohighlight" title="JWT payload of a JWT-based verifiable credential using JWS as a proof (non-normative)">
+            {
+              "sub": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+              "jti": "http://dmv.example.gov/credentials/3732",
+              "iss": "did:example:abfe13f712120431c276e12ecab",
+              "iat": "1541493724",
+              "exp": "1573029723",
+              "nonce": "660!6345FSer",
+              "vc": {
+                "@context": [
+                  "https://w3.org/2018/credentials/v1",
+                  "https://example.com/examples/v1"
+                ],
+                "type": ["VerifiableCredential", "ProofOfAgeCredential"],
+                "credential_subject": {
+                  "age_over": 21
+                }
+              }
+            }
+            </pre>
+
+            <p class="issue">
+                Add a short description explaining the example above. Also mention that @context might be omitted.
+            </p>
+
+            <p class="issue">
+                Add an example of how a base64 JWT encoded verifiable credential looks like.
+            </p>
+
+            <pre class="example nohighlight" title="JWT header of a JWT based verifiable presentation (non-normative)">
+            {
+              "alg": "RS256",
+              "typ": "JWT",
+              "kid": "did:example:ebfeb1f712ebc6f1c276e12ec21#keys-1"
+            }
+            </pre>
+
+            <p class="issue">
+                Add a short description explaining the example above.
+            </p>
+
+            <pre class="example nohighlight" title="JWT payload of a JWT based verifiable presentation (non-normative)">
+            {
+              "iss": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+              "jti": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
+              "aud": "did:example:4a57546973436f6f6c4a4a57573",
+              "iat": "1541493724",
+              "exp": "1573029723",
+              "nonce": "343§$FSFDa-",
+              "vcp": {
+                "@context": [
+                  "https://w3.org/2018/credentials/v1",
+                  "https://example.com/examples/v1"
+                ],
+                "type": ["VerifiablePresentation"],
+                "vc_jwt": ["..."]
```

Comment:
`vc_jwt` should be `verifiableCredential`.

Reply:
agree